pkg/kubectl/resource/helper.go

```diff
@@ -63,14 +63,19 @@ func (m *Helper) Get(namespace, name string, export bool) (runtime.Object, error
 	return req.Do().Get()
 }
 
-// TODO: add field selector
-func (m *Helper) List(namespace, apiVersion string, selector string, export, includeUninitialized bool) (runtime.Object, error) {
+func (m *Helper) List(namespace, apiVersion, labelSelector, fieldSelector string, export, includeUninitialized bool) (runtime.Object, error) {
```

Comment:
this should take ListOptions, like https://github.com/kubernetes/kubernetes/pull/53768/files?w=1#diff-df181bee3d6c868b2cb758df5a1b5226R66